content/faq.md

```diff
@@ -55,51 +76,82 @@ It’s also worth noting that, while there are almost 8B people on Earth, there
 
 ### Why do you use Ethereum? {#why-eth}
 
-As of 2019, Ethereum is the most widely deployed, most secure, and best documented general purpose blockchain. Using Ethereum is a practical engineering choice. It’s the best way to bootstrap real cryptographic ownership. We’re not specifically interested in Ethereum one way or the other.
+As of 2019 (when [Azimuth](#what-is-azimuth) was launched), Ethereum is the most widely deployed,
```

Comment:
Mix of tenses feels weird here? Maybe just me.

Reply:
https://github.com/urbit/urbit.org/pull/538/commits/2786fcef5920296d3d9b12d1c37b1bf49d2b4717